conans/test/functional/generators/components/test_components_cmake_generators.py

```diff
@@ -50,12 +50,16 @@ def package_info(self):
                 elif self.options.components == "custom":
                     self.cpp_info.filenames["cmake_find_package_multi"] = "MYG"
                     self.cpp_info.filenames["cmake_find_package"] = "MYG"
+                    self.cpp_info.filenames["CMakeDeps"] = "MYG"
```

Comment:
This might not be necessary, there is an adaptor to use ``cmake_find_package_multi`` definition in ``CMakeDeps`` too (necessary for painless migration)